ICAFIX/ReApplyFixMultiRunPipeline.sh

```diff
@@ -689,32 +642,15 @@ case ${MatlabRunMode} in
 	0)
 		# Use Compiled Matlab
 
-		matlab_exe="${FSL_FIXDIR}/compiled/$(uname -s)/$(uname -m)/run_fix_3_clean.sh"
+		matlab_exe="${FSL_FIXDIR}/compiled/$(uname -s)/$(uname -m)/run_fix_3_clean.sh" # this needs to be changed
```

Comment:
Is this commit incomplete?

Reply:
Yes, I think he left that comment because it needs to change to a pipelines-internal compiled version of `fix_3_clean`, while it currently points to a legacy fix installation folder structure.